ex03.exs

```diff
@@ -50,9 +50,22 @@ defmodule Ex03 do
      it is nicely structured            7
   """
 
+  def map(list, function) do
+    Enum.map(list,function)
+  end
+
+  def separate(collection, process_count, function) do
+    temp = [] ++ [collection]
+    list = Enum.flat_map(temp, &(&1))
+
+    Enum.chunk_every(list, ceil(length(list)/process_count)) 
```

Comment:
You are the _only_ person in the class to use `ceil` and not `trunc`! 

Thank you.

That one line of code means you got some points for this section:

# 10